packages/pigeon/lib/java_generator.dart

```diff
@@ -815,9 +859,12 @@ String _javaTypeForDartType(TypeDeclaration type) {
   return _javaTypeForBuiltinDartType(type) ?? type.baseName;
 }
 
+String _nullSafeString(TypeDeclaration type) {
```

Comment:
It wasn't clear to me what this was doing from the name at the call site earlier. How about "_typeWithNullabilityAnnotation(...)"?

Reply:
how about `nullabilityAnnotationFromType()`?

Reply:
I would expect that to return just the string `@Nullable` or `@Nonnull` (which is the annotation), rather than the type with its annotation.

Reply:
That's exactly what it does

Reply:
🤦🏻 Somehow I thought it was adding it to the type and returning that. Your name SGTM.